### PR TITLE
Feat/add&remove wallet

### PR DESCRIPTION
**Pull request by pitb2022**

### CODE REVIEW

File: src/components/Dialogs/RemoveWalletLoginDialog/gql.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import gql from 'graphql-tag'
+
+export const REMOVE_WALLET_LOGIN = gql`
+  mutation RemoveWalletLogin {
+    removeWalletLogin {
+      id
```
Comment (robertu7) [inline src/components/Dialogs/RemoveWalletLoginDialog/gql.ts:6]: try to query `socialAccounts` here so we can skip to refetch [`ROOT_QUERY_PRIVATE`](https://github.com/thematters/matters-web/pull/3800/files#diff-0a30d104691d84a6763d818bc6d806388ad0a26ea49081996fe3cb2a734f7b2aR38):


```
    removeWalletLogin {
      id
      socialAccounts { ... }
    }
```

File: src/components/Forms/WalletAuthForm/Connect.tsx [hunk 2]
```diff
@@ -156,41 +172,72 @@ const Connect: React.FC<FormProps> = ({
...
+            })
+          } else if (submitCallback) {
+            submitCallback(loginData?.walletLogin.type)
+          }
+        }
+
+        if (isConnect) {
+          await addWalletLogin({
+            variables: variables,
+            // FIXME: Why automatically update?
```
Comment (robertu7) [inline src/components/Forms/WalletAuthForm/Connect.tsx:213]: fixed or not?

File: src/components/Forms/WalletAuthForm/gql.ts [hunk 3]
```diff
@@ -29,3 +29,14 @@ export const WALLET_LOGIN = gql`
     }
   }
 `
+
+export const ADD_WALLET_LOGIN = gql`
+  mutation AddWalletLogin($input: WalletLoginInput!) {
+    addWalletLogin(input: $input) {
+      id
+      info {
+        ethAddress
```
Comment (robertu7) [inline src/components/Forms/WalletAuthForm/gql.ts:38]: same as above, can add fields to query for local data updating

**Review by robertu7 — APPROVED** [review]

File: src/components/Dialogs/RemoveWalletLoginDialog/gql.ts (hunk 1 above)
Comment (pitb2022) [reply src/components/Dialogs/RemoveWalletLoginDialog/gql.ts:6]: query user.info.ethAddress?

File: src/components/Forms/WalletAuthForm/Connect.tsx (hunk 2 above)
Comment (pitb2022) [reply src/components/Forms/WalletAuthForm/Connect.tsx]: fixed

**Pull request merged**